src/test/regress/pg_regress_multi.pl

```diff
@@ -56,6 +60,10 @@ ()
 my %functions = ();
 my %operators = ();
 my $valgrind = 0;
+my $valgrind_path = "valgrind";
```

Comment:
I think it is not possible to have a variable name with hyphen "-" in perl. General convention for command line parameters is to separating words with hyphen. We just map hyphened variables to underscored variables below.